microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -904,30 +948,98 @@ See also:
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 125.
-* This checkbox is present in Microsoft Edge Stable 124.
+* This checkbox is not present in Microsoft Edge Canary 128.
+* This checkbox is present in Microsoft Edge Stable 125.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable user timings based extensions
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new label for checkbox as of Microsoft Edge 128_
+
+Enables extensions that customize the appearance of user timing events in the **Performance** tool.
+
+See also:
+* [Performance features reference](../evaluate-performance/reference.md)
+* [Performance recording event reference](../evaluate-performance/performance-reference.md)
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 128.
+* This checkbox is present in Microsoft Edge Stable 125.
 
 
 <!-- ====================================================================== -->
 ## Timeline: Enable User Timings based extensions
-<!-- no (?) link -->
+<!-- checkbox has no (?) link pointing to this anchor wording -->
 
 _new as of Microsoft Edge 125_
 
-Enables extensions that customize the appearance of user timing events in the **Performance** tool.
+See the above section, showing the newer label for the checkbox.
+
+
+<!-- ====================================================================== -->
+## Performance panel: Enable collecting enhanced traces
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new as of Microsoft Edge 128_
+
+In the **Performance** tool, when recording, collects enhanced traces.
+<!-- todo: real desc - maybe extract from article -->
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 128.
+* This checkbox is not present in Microsoft Edge Stable 125.
 
 See also:
+* [Share enhanced performance and memory traces](../experimental-features/share-traces.md)
 * [Performance features reference](../evaluate-performance/reference.md)
 * [Performance recording event reference](../evaluate-performance/performance-reference.md)
 
+
+<!-- ====================================================================== -->
+## Performance panel: Enable collecting source text for compiled script
+<!-- checkbox has no (?) link pointing to this anchor wording -->
+
+_new as of Microsoft Edge 128_
+
+In the **Performance** tool, when recording, collects source text (the original source code) for compiled JavaScript code.
+<!-- todo: real desc -->
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 128.
+* This checkbox is not present in Microsoft Edge Stable 125.
+
+See also:
+* [Share enhanced performance and memory traces](../experimental-features/share-traces.md)
+* [Performance features reference](../evaluate-performance/reference.md)
+* [Performance recording event reference](../evaluate-performance/performance-reference.md)
+<!-- todo: best articles? -->
```

Comment:
best See Also's?
@captainbrosset

Reply:
I think the article about sharing enhanced traces is the best See Also we can add here.